### DISCUSSION TITLE
No validation error when a file exceeds 30Mo

### DISCUSSION THREAD

**Discussion by mlcpro** [symfony/symfony#49903] [Q&A]
```
  $builder
            ->add('name', FileType::class, [
                'label' => false,
                'mapped' => false,
                'required' => false,
                'constraints' => [
                    new NotNull(),
                    new Image([
                        'maxSize' => '5M',
                    ])
                ]
            ])
        ;
```
I use the image constraint, to do tests I try to send a .zip file with a size of 30Mo, the form submits and an unexpected behavior this product, no validation error is returned, and when I tries to trap the error in a condition like `if($form->isSubmitted() && !$form->isValid())` the browser seems to be looping.

ideas ?

**Comment by ghost** [comment]
hard to say. `dd()` inside `\Symfony\Component\Validator\Constraints\ImageValidator::validate` and check if it is even executed (maxSize validation takes place inside a parent class - FileValidator, so check this too)

BTW. change Image constraint to File. It's semantically more correct as you upload zip file.
